Application class not found in version 0.6.0 container

Docker logs shows this:
```
Error: Could not find or load main class com.penguineering.cleanuri.Application
Caused by: java.lang.ClassNotFoundException: com.penguineering.cleanuri.Application
```

--- a/pom.xml
+++ b/pom.xml
@@ -21,7 +21,7 @@
     <micronaut.runtime>netty</micronaut.runtime>
     <micronaut.aot.enabled>false</micronaut.aot.enabled>
     <micronaut.aot.packageName>com.penguineering.cleanuri.aot.generated</micronaut.aot.packageName>
-    <exec.mainClass>com.penguineering.cleanuri.Application</exec.mainClass>
+    <exec.mainClass>com.penguineering.cleanuri.extractor.Application</exec.mainClass>
   </properties>
 
   <repositories>

